subprojects/docs/src/docs/userguide/jvm/jvm_test_suite_plugin.adoc

```diff
@@ -153,13 +151,37 @@ To use these and similar methods, you need to access the top-level dependencies
 Though it is discouraged, as suggested by the comment in the snippet above, you can also access the configurations of a test suite in a top-level `dependencies` block after creating the test suite.  The configurations used by test suites are normal Gradle configurations, but their exact names should be considered an implementation detail subject to change.  For this reason, configuration via the suite's own `dependencies` block is recommended.
```

Comment:
Could we also make this language even stronger here?  Maybe we could also warn upon doing this?

Reply:
I feel like this is a separate PR.